JCenter kills doctoolchain, as Grolifant is not available on Maven Central

**Describe the bug**
Since some days, I am unable to build our CI/CD image, as the dependency `org.ysb33r.gradle:grolifant:0.12.1` is no longer available.

```
#35 [stage-1 26/31] RUN JAVA_HOME=/opt/dtcw/.doctoolchain/jdk/ /opt/dtcw/.doctoolchain/docToolchain-2.2.1/bin/doctoolchain .     exportOpenApi
#35 sha256:1555dd44d7cf8a3ed225331f5639d541944d38400557563e367e1ad02f429470
#35 3.599
#35 3.599 Welcome to Gradle 6.9.2!
#35 3.599
#35 3.601 Here are the highlights of this release:
#35 3.601  - This is a small backport release.
#35 3.602  - Java 16 can be used to compile when used with Java toolchains
#35 3.603  - Dynamic versions can be used within plugin declarations
#35 3.603  - Native support for Apple Silicon processors
#35 3.603
#35 3.604 For more details see https://docs.gradle.org/6.9.2/release-notes.html
#35 3.604
#35 4.436 Starting a Gradle Daemon (subsequent builds will be faster)
#35 64.54
#35 64.54 FAILURE: Build failed with an exception.
#35 64.54
#35 64.54 * What went wrong:
#35 64.54 A problem occurred configuring root project 'docToolchain'.
#35 64.54 > Could not resolve all artifacts for configuration ':classpath'.
#35 64.54    > Could not find org.ysb33r.gradle:grolifant:0.12.1.
#35 64.54      Searched in the following locations:
#35 64.54        - https://plugins.gradle.org/m2/org/ysb33r/gradle/grolifant/0.12.1/grolifant-0.12.1.pom
#35 64.54      If the artifact you are trying to retrieve can be found in the repository but without metadata in 'Maven POM' format, you need to adjust the 'metadataSources { ... }' of the repository declaration.
#35 64.54      Required by:
#35 64.54          project : > org.asciidoctor.convert:org.asciidoctor.convert.gradle.plugin:2.4.0 > org.asciidoctor:asciidoctor-gradle-jvm:2.4.0
#35 64.54          project : > org.asciidoctor.convert:org.asciidoctor.convert.gradle.plugin:2.4.0 > org.asciidoctor:asciidoctor-gradle-jvm:2.4.0 > org.asciidoctor:asciidoctor-gradle-base:2.4.0
#35 64.54
#35 64.54 * Try:
#35 64.54 Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
#35 64.55
#35 64.55 * Get more help at https://help.gradle.org
#35 64.55
#35 64.55 BUILD FAILED in 1m 3s
#35 ERROR: executor failed running [/bin/sh -c JAVA_HOME=/opt/dtcw/.doctoolchain/jdk/ /opt/dtcw/.doctoolchain/docToolchain-2.2.1/bin/doctoolchain .     exportOpenApi]: exit code: 1
```

Looking at `https://plugins.gradle.org/m2/org/ysb33r/gradle/grolifant/` you will get the message, that _JCenter no longer serves files but redirects to Maven..._. But on Maven Central is only version `0.16.1` available.

**To Reproduce**
Steps to reproduce the behavior:
1. Use a fresh install of docToolchain
2. Run the task exportOpenAPI
4. See error

**Expected behavior**
Task `exportOpenAPI` succeeds, as all dependencies are available

--- a/build.gradle
+++ b/build.gradle
@@ -1,5 +1,24 @@
 import java.util.zip.ZipFile
 
+// fix versions for #1200
+buildscript {
+    configurations.all {
+        resolutionStrategy.eachDependency { DependencyResolveDetails details ->
+            if (details.requested.group == 'org.ysb33r.gradle' && details.requested.name == 'grolifant' && details.requested.version == '0.12.1') {
+                details.useVersion '0.16.1'
+                details.because 'jcenter redirect'
+                System.out.println ">>>>>>>>> fix grolifant"
+            }
+            if (details.requested.group == 'org.codehaus.groovy.modules.http-builder' && details.requested.name == 'http-builder' && details.requested.version == '0.7.2') {
+                details.useVersion '0.7.1'
+                details.because 'jcenter redirect'
+                System.out.println ">>>>>>>>> fix http-builder"
+            }
+        }
+    }
+}
+// /fix versions for #1200
+
 /*
  * This build file is part of the docToolchain
  */

